### PR TITLE
sync up runtimes

### PR DESCRIPTION
**Pull request by saraswatpuneet**
# Goal
The goal of this PR is to sync up runtimes in preparation to #294 which will be done on the side

Closes: #305 as a side effect

### CODE REVIEW

**Review by shannonwells — APPROVED** [review]
Should we be doing development in frequency-rococo first, and then move things to frequency?

**Review by enddynayn — COMMENTED** [review]
this might be a good opportunity to refactor the common things.

**Review by enddynayn — APPROVED** [review]

File: runtime/frequency-rococo/Cargo.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ log = { version = "0.4.17", default-features = false }
 scale-info = { version = "2.1.1", default-features = false, features = ["derive"] }
 serde = { version = "1.0.137", optional = true, features = ["derive"] }
 smallvec = "1.6.1"
+getrandom = { version = "0.2", default-features = false, features = ["js"] }
```
Comment (aramikm) [inline runtime/frequency-rococo/Cargo.toml:24]: I'm curious which pallet or part of runtime code needs randomness?

Comment (saraswatpuneet) [reply runtime/frequency-rococo/Cargo.toml:24]: I am too I just copied frequency to rococo but will find out from Shannon

**Pull request merged**